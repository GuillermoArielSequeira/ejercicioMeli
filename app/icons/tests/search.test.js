const React = require('react');
const { shallow } = require('enzyme');
const Search = require('../serach');

describe('Search svg Component', () => {
  it('render Search correctly', () => {
    const wrapper = shallow(<Search />);
    expect(wrapper).toMatchSnapshot();
  })
});
