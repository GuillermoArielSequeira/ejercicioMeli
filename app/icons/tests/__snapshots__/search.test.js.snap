// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search svg Component render Search correctly 1`] = `ShallowWrapper {}`;
